lib/simditor.js

```diff
@@ -503,7 +498,8 @@ Formatter = (function(superClass) {
       h1: ['margin-left', 'text-align'],
       h2: ['margin-left', 'text-align'],
       h3: ['margin-left', 'text-align'],
-      h4: ['margin-left', 'text-align']
+      h4: ['margin-left', 'text-align'],
+      input: ['width']
```

Comment:
I think this is not necessary now since we are resolving the width of the input via *style* attr.

Reply:
Yes it is, I tested and it doesn't work if you don't specify the actual prop of the style